com.doycho.euterpe.gtk.json

```diff
@@ -0,0 +1,66 @@
+{
+    "app-id" : "com.doycho.euterpe.gtk",
+    "runtime" : "org.gnome.Sdk",
```

Comment:
do you mean to be using the Sdk as the runtime? This is more normally org.gnome.Platform (unless you explicitly need dev tools at runtime)

Reply:
Good catch. I totally did not notice that when submitting. Changed the runtime to "org.gnome.Platform".